eth/eventhandler/event_handler_test.go

```diff
@@ -577,9 +596,11 @@ func setupEventHandlerWithMockedCtrl(t *testing.T, ctx context.Context, logger *
 		Ctx: ctx,
 	})
 	require.NoError(t, err)
+	_, operatorSk, err := rsaencryption.GenerateKeys()
+	require.NoError(t, err)
 
 	storageMap := ibftstorage.NewStores()
-	nodeStorage, _ := setupOperatorStorage(logger, db)
+	nodeStorage, _ := setupOperatorStorage(logger, db, &testOperator{priv: operatorSk, id: 0})
```

Comment:
if we would ever want the storage to have an operator in test that use `setupEventHandlerWithMockedCtrl`. we would probably need to change this. also 0 is not a valid operator key.
if these test didn't have valid operator to this day we will need to add in the future.